### ISSUE TITLE
[meizu 16th] sound

### ISSUE THREAD

**Issue by Pa4ek24** [phhusson/treble_experimentations#783]
The sound does not work.  Absolutely.  There is no way to answer the call due to idle sound

**Comment by phhusson** [comment]
Your message seem to imply that multimedia sound works...?
Is that on my v119?

**Comment by Pa4ek24** [comment]
No, the sound does not work at all, neither multimedia, nor anything else.  even the sound in dialer does not work.  this is on v200.d

**Comment by phhusson** [comment]
what is the vendor android version?
Does it work in my pie GSI?
Get me logcat on boot

**Comment by Pa4ek24** [comment]
vendor 8.1, but now stands HavocOS on pie.  your gsi 9 did not try.  where can I find?

**Comment by lanreallenjay** [comment]
This same issue is happening on Motorola Moto g6 arm 32 with pie vendor. I reported it here earlier and I provided a dmesg log

**Comment by Lorondos** [comment]
Same issue on my BV9600 Pro. No audio at all, Speaker nor Bluetooth Audio.
